cmds/info.c

```diff
@@ -851,6 +851,41 @@ static int read_uftrace_version(void *arg)
 	return 0;
 }
 
+static int fill_utc_offset(void *arg)
+{
+	struct fill_handler_arg *fha = arg;
+	time_t current_time;
+	struct timespec ts;
+	double offset;
+
+	time(&current_time);
+	clock_gettime(clock_source, &ts);
+
+	/* calculate time offset between UTC and clock_source. */
+	offset = difftime(current_time, ts.tv_sec);
+
+	dprintf(fha->fd, "utc_offset:%ld\n", (time_t)offset);
```

Comment:
For `%ld`, you'd better cast it to `unsigned long`.

Reply:
As I understand, `time_t` is `long` type so `%ld` is used instead of `%lu`.
```i
# 143 "/usr/include/x86_64-linux-gnu/bits/types.h" 2 3 4
        ...
typedef long int __time_t;
        ...
typedef __time_t time_t;
```
Since this `offset` is used to adjust the other `time_t` value for UTC time, I made it casted to the same type `time_t`. Should I change it to `unsigned long` and print it with `%lu` instead?

Reply:
Oh, sorry.  I meant `long`.  But it could be different on different architecture or distro.  So I think it's better to explicitly cast to `long` instead of `time_t`.

Reply:
OK. I've just changed it to use `long` for `offset` type itself.